python/src/opendp/_extrinsics/microdata/universal_microdata_scheme.py

```diff
@@ -0,0 +1,166 @@
+from typing import Callable, Union, Optional
+
+from opendp.mod import Domain, Metric, Measurement
+import opendp.prelude as dp
+from opendp._lib import get_np_csprng, import_optional_dependency
+from opendp._extrinsics._make_private_selection import make_private_selection_threshold
+from opendp._extrinsics.microdata.faithfulness import make_private_faithfulness
+from opendp._extrinsics.synth import make_private_synthesizer_trainer
+
+
+__all__ = ["make_private_universal_microdata_scheme"]
+
+
+def make_private_universal_microdata_scheme(input_domain: Domain,
+                                            input_metric: Metric,
+                                            synthesizer_epsilon: float,
+                                            configuration_candidates: Optional[list[dict]] = None,
+                                            quality_acceptance_criteria: Optional[list[tuple]] = None,
+                                            quality_epsilons: Optional[list[float]] = None,
```

Comment:
Sometimes for list args, rather than making it optional, it can be simpler to make the default `[]`, so you don't need to handle `None` as a special case... but if you actually want those to work differently, ignore me!

Reply:
(assuming you don't mutate them, as default args are shared objects across function invocations in Python)